weekly-mission(week2)/src/features/signUp.js

```diff
@@ -1,8 +1,13 @@
-import { addErrorMsgToMismatchConfirm } from "../view/addErrorMessageToInput.js";
+import { signUp } from "../apis/signupUser.js";
+import { emailInput, passwordInput } from "../constants/dom.js";
 
 export const signUpHandler = (e) => {
   e.preventDefault();
-  addErrorMsgToMismatchConfirm();
+  
+  const inputAccount = {
```

Comment:
요정도로 작성되면 괜찮을 것 같아요!
```suggestion
  const account = {
```